xdp-dump/xdpdump.c

```diff
@@ -924,6 +924,20 @@ static int match_target_function(struct dumpopt *cfg,
 	return -EAGAIN;
 }
 
+
+/*****************************************************************************
+ * check_btf()
+ *****************************************************************************/
+static bool check_btf(struct xdp_program *prog)
+{
+	if (xdp_program__btf(prog))
+		return true;
+
+	pr_warn("ERROR: The loaded XDP program has no BTF information, "
+		"which xdpdump requires!\n");
```

Comment:
Small nit: this could be interpreted to mean "xdpdump requires that there is no BTF information". So maybe change it to something like "xdpdump requires BTF information, but that is missing from the loaded XDP program"?

Reply:
When reading again you are right will update pull req.